app/views/comments/_comment.html.erb

```diff
@@ -1,7 +1,7 @@
 
 <p>
   <strong>Commenter:</strong>
-  <%= comment.commenter %>
+  <%= comment.user_id %>
```

Comment:
Display name of user, who added a comment.

Reply:
done